pkg/plugins/election/elector.go

```diff
@@ -65,7 +65,8 @@ func StartElection(conf *config.Config) Election {
 
 	err = elector.StartLeading(context.Background())
 	if err != nil {
-		log.Logger.Info("msg", "try get leader lock failed, server is not leader", "elector", conf.Elector, "err", err)
+		log.Logger.Error("msg", "try get leader lock failed, server is not leader", "elector", conf.Elector, "err", err)
+		return nil
```

Comment:
not certainly.. because `start leading failed` brings 2 scenes:
1. activate `elector leading` , but there are some error when connect to `k8s` or `redis`, so leading failed.
2. try to become a leader , but other server is leader now so current server lock failed.

both of this 2 conditions will return error. maybe should add more `if` to find out the real error.

Reply:
It doesn't matter. Fail fast. We'll use metrics to express the status later.